openmetadata-service/src/test/java/org/openmetadata/service/alerts/AlertsRuleEvaluatorTest.java

```diff
@@ -3,138 +3,165 @@
+    CreateTable create = tableResourceTest.createRequest(test).withColumns(columns);
+    Table createdTable = tableResourceTest.createAndCheckEntity(create, ADMIN_AUTH_HEADERS);
+    // Create a change Event
+    ChangeEvent changeEvent = new ChangeEvent();
+    changeEvent.setEntityType(Entity.TABLE);
+    changeEvent.setEntity(createdTable);
+
+    // Check AlertRuleEvaluator
+    AlertsRuleEvaluator alertsRuleEvaluator = new AlertsRuleEvaluator(changeEvent);
+    EvaluationContext evaluationContext = new StandardEvaluationContext(alertsRuleEvaluator);
+    String fqn = createdTable.getFullyQualifiedName();
+    assertTrue(evaluateExpression("matchAnyEntityFqn('" + fqn + "')", evaluationContext));
+    assertFalse(evaluateExpression("matchAnyEntityFqn('testFQN1')", evaluationContext));
   }
 
   @Test
-  void test_matchAnyTag() {
-    table.withTags(getTags("tag1", "tag2", "tag3"));
-
-    // Tag is present
-    assertTrue(evaluateExpression("matchAnyTag('tag1', 'tag2', 'tag3', 'tag4')"));
-    assertFalse(evaluateExpression("!matchAnyTag('tag1', 'tag2', 'tag3', 'tag4')"));
-    assertTrue(evaluateExpression("matchAnyTag('tag1', 'tag2', 'tag4')"));
-    assertFalse(evaluateExpression("!matchAnyTag('tag1', 'tag2', 'tag4')"));
-    assertTrue(evaluateExpression("matchAnyTag('tag1', 'tag2', 'tag4')"));
-    assertFalse(evaluateExpression("!matchAnyTag('tag1', 'tag2', 'tag4')"));
-    assertTrue(evaluateExpression("matchAnyTag('tag1', 'tag4')"));
-    assertFalse(evaluateExpression("!matchAnyTag('tag1', 'tag4')"));
-
-    // Tag `tag4` is not present
-    assertFalse(evaluateExpression("matchAnyTag('tag4')"));
-    assertTrue(evaluateExpression("!matchAnyTag('tag4')"));
+  void test_matchAnyEntityId(TestInfo test) throws IOException {
+    // Create Table
+    List<Column> columns = List.of(new Column().withName("c1").withDataType(ColumnDataType.INT));
+    CreateTable create = tableResourceTest.createRequest(test).withColumns(columns);
+    Table createdTable = tableResourceTest.createAndCheckEntity(create, ADMIN_AUTH_HEADERS);
+    // Create a change Event
+    ChangeEvent changeEvent = new ChangeEvent();
+    changeEvent.setEntityType(Entity.TABLE);
+    changeEvent.setEntity(createdTable);
+
+    // Check AlertRuleEvaluator
+    AlertsRuleEvaluator alertsRuleEvaluator = new AlertsRuleEvaluator(changeEvent);
+    EvaluationContext evaluationContext = new StandardEvaluationContext(alertsRuleEvaluator);
+    String id = createdTable.getId().toString();
+    assertTrue(evaluateExpression("matchAnyEntityId('" + id + "')", evaluationContext));
+    assertFalse(evaluateExpression("matchAnyEntityId('" + UUID.randomUUID() + "')", evaluationContext));
   }
 
-  private Boolean evaluateExpression(String condition) {
-    return parseExpression(condition).getValue(evaluationContext, Boolean.class);
+  @Test
+  void test_matchAnyEventType() {
+    // Create a change Event
+    ChangeEvent changeEvent = new ChangeEvent();
```

Comment:
Comments like Create a change Event, is not helpful. Please add information on what you are doing.

Reply:
updated